cdc/generation.cc

```diff
@@ -718,58 +718,67 @@ future<> generation_service::check_and_repair_cdc_streams() {
         }
     }
 
-    bool should_regenerate = false;
-    std::optional<topology_description> gen;
-
-    static const auto timeout_msg = "Timeout while fetching CDC topology description";
-    static const auto topology_read_error_note = "Note: this is likely caused by"
-            " node(s) being down or unreachable. It is recommended to check the network and"
-            " restart/remove the failed node(s), then retry checkAndRepairCdcStreams command";
-    static const auto exception_translating_msg = "Translating the exception to `request_execution_exception`";
-    const auto tmptr = _token_metadata.get();
+    auto tmptr = _token_metadata.get();
```

Comment:
Why not const any more?

Reply:
because it's moved out from later in the function